src/main/java/org/osiam/resources/controller/ServiceProviderConfigsController.java

```diff
@@ -50,7 +48,7 @@ public ServiceProviderConfig getConfig() {
         public static final String SCHEMA = "urn:ietf:params:scim:schemas:core:2.0:ServiceProviderConfig";
         public final Supported patch = new Supported(true);
         public final Supported bulk = new BulkSupported(false);
-        public final Supported filter = new FilterSupported(true, RequestParamHelper.MAX_RESULT);
+        public final Supported filter = new FilterSupported(true, 100);
```

Comment:
Should be configurable anyways, but that's out of scope. AFAIK we do not even enforce this limit on the service level. 

@dacrome: where should this constant belong? `RequestParamHelper` does not exist anymore.